operator-common/src/main/java/io/strimzi/operator/common/model/Ca.java

```diff
@@ -778,6 +761,13 @@ public int certGeneration() {
         return INIT_GENERATION;
     }
 
+    /**
+     * @return the generation of the current CA certificate as an annotation
+     */
+    public Map<String, String> caCertGenerationFullAnnotation() {
```

Comment:
I've updated to change it to Map.Entry. At the moment all of the clustered components (Cruise Control, ZooKeeper, Kafka) call ModelUtils directly to create certificate Secrets. The other components with a single pod call buildTrustedCertificateSecret which is in CertUtils. So that's why there are call from the different places.